app/web/i18n/server-side-translations.ts

```diff
@@ -0,0 +1,17 @@
+import { GetStaticProps } from "next";
+import nextI18NextConfig from "next-i18next.config";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+
+export const serverSideTranslationProps = async (
+  locale: string | undefined,
+  namespaces: Array<string>
+) =>
+  await serverSideTranslations(locale ?? "en", namespaces, nextI18NextConfig);
+
+export const translationStaticProps =
+  (namespaces: Array<string>): GetStaticProps =>
+  async ({ locale }) => ({
+    props: {
+      ...serverSideTranslationProps(locale, namespaces),
```

Comment:
```suggestion
      ...(await serverSideTranslationProps(locale, namespaces)),
```
I think it might be because this is not being awaited?

Reply:
Oh, I am really sorry about that. I'll test more carefully and update the PR.

Reply:
Yes, that was it. It is fixed now